CMakeLists.txt

```diff
@@ -730,6 +730,7 @@ install(TARGETS ${MXNET_INSTALL_TARGETS}
 )
 
 install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(DIRECTORY 3rdparty/tvm/nnvm/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
Isn't mshadow needed as well?

Reply:
As far as I could tell, no public headers include it. At least, my C++ project importing the full cpp-package header works without mshadow being installed.